Examples/Swift/AnimatedLineExample.swift

```diff
@@ -40,13 +40,11 @@ class AnimatedLineExample_Swift: UIViewController, MGLMapViewDelegate {
 
         // Add a layer to style our polyline.
         let layer = MGLLineStyleLayer(identifier: "polyline", source: source)
-        layer.lineJoin = MGLStyleValue(rawValue: NSValue(mglLineJoin: .round))
-        layer.lineCap = MGLStyleValue(rawValue: NSValue(mglLineCap: .round))
-        layer.lineColor = MGLStyleValue(rawValue: UIColor.red)
-        layer.lineWidth = MGLStyleFunction(interpolationMode: .exponential,
-            cameraStops: [14: MGLConstantStyleValue<NSNumber>(rawValue: 5),
-                          18: MGLConstantStyleValue<NSNumber>(rawValue: 20)],
-            options: [.defaultValue : MGLConstantStyleValue<NSNumber>(rawValue: 1.5)])
+        layer.lineJoin = NSExpression(forConstantValue: NSValue(mglLineJoin: .round))
+        layer.lineCap = NSExpression(forConstantValue: NSValue(mglLineJoin: .round))
+        layer.lineColor = NSExpression(forConstantValue: UIColor.red)
+        layer.lineWidth = NSExpression(format: "mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)",
```

Comment:
Additional comments here would be useful to explain what the lineWidth expression is doing.